app/translations/dutch.js

```diff
@@ -0,0 +1,396 @@
+  sidebarSend: 'Verzenden',
+  sidebarReceive: 'Ontvangen',
+  sidebarContacts: 'Contacten',
+  sidebarTokenSale: 'Token Verkoop',
+  sidebarNews: 'Nieuws',
+  sidebarSettings: 'Instellingen',
+  sidebarLogout: 'Uitloggen',
+  sidebarCurrentBlock: 'HUIDIG BLOK',
+}
+
+const MANAGE_WALLETS = {
+  manageWalletsLabel: 'Beheer wallets',
+  manageWalletsImport: 'Importeren',
+  manageWalletsCreate: 'Maak',
+  manageWalletsEdit: 'Aanpassen',
+  manageWalletsEditWallet: 'Pas wallet aan',
+  manageWalletsEditWalletInstructions: 'Wijzig details',
+  manageWalletsEditWalletNameLabel: 'WALLET NAAM',
+  manageWalletsEditWalletNamePlaceholder: 'Wallet naam',
+  manageWalletsEditWalletAddressLabel: 'WALLET ADRES',
+  manageWalletsEditWalletSave: 'Wijzigingen opslaan',
+}
+
+const ACTIVITY = {
+  activityAddAddress: 'Toevoegen',
+  activityViewTx: 'Bekijken',
+  activityPageLabel: 'Alle activiteiten',
+  activityExport: 'Exporteren',
+}
+
+const RECEIVE = {
+  recieveSelectMethod: 'Selecteer stortingsmethode',
+  receiveAssetsAddressLabel: 'Jouw publieke adres',
+  receivePageLabel: 'Ontvang assets',
+  receiveYourAddressTabLabel: 'JOUW ADRES',
+  receiveCopyCodeButton: 'Copy Code Afbeelding',
+  receiveDisclaimer:
+    'Stuur alleen assets die <b>compatibel zijn met de NEO-blockchain (NEO, GAS, etc.)</b>. Het verzenden van andere assets zal resulteren in permanent verlies.',
+  receiveRequestTabAssets: 'VERZOEK ASSETS',
+  recieveWhyUseQRLabel: 'Waarom een QR-Code gebruiken??',
+  receiveQRExplanation:
+    '<p>Ooit assets naar het verkeerde adres gestuurd vanwege een foutief karakter in het wallet-adres?</p><p>Zo niet, dan heb je mazzel - maar het gebeurt veel te vaak.</p><p>Hier bij CoZ willen we ervoor zorgen dat mensen die jou betalen, het juiste adres krijgen. J kunt een QR-code genereren om assets aan te vragen, zodat ze jou kunnen helpen. </p> <p> Elke code die je genereert, bevat uw openbare wallet-adres, een aantal en een referentie - allemaal door jou ingesteld. </p>',
+}
+
+const REQUEST_ASSETS = {
+  requestAssetLabel: 'ASSET',
+  requestAssetAmountLabel: 'Aantal',
+  requestAssetAmount: 'AANTAL',
+  requestAssetDepositLabel: 'SORT IN DEZE WALLET',
+  requestAssetRefLabel: 'REFERENTIE',
+  requestAssetRefPlaceholder: 'Voeg een notitie toe...',
+  requestAssetQRButton: 'Genereer QR-Code',
+  requestAssetYourQRHeader: 'Jou QR Code',
+  requestAssetsPaymentDetails: 'BETAALVERZOEK DETAILS',
+  requestAssetsYourQRLabel: 'Jouw QR-Code',
+  requestAssetsRefLabel: 'REFERENTIE',
+  requestAssetsAddressLabel: 'ADRES',
+  requestAssetsAmountLabel: 'AANTAL',
+  requestAssetsAssetLabel: 'ASSET',
+}
+
+const TRANSACTION_FEE = {
+  transactionFeeQuestion: 'PRIORITEER JOUW TRANSACTIE MET EEN FEE?',
+  fast: 'Snel',
+  faster: 'Sneller',
+  fastest: 'Snelst',
+  sendWithFee: 'Verstuur {itemCount, plural, one {Asset} other {Assets}} Met fee',
+  sendWithoutFee:
+    'Verstuur {itemCount, plural, one {Asset} other {Assets}} Zonder fee',
```

Comment:
"one", "other"

Reply:
In the german.js is the following line:

sendWithFee:
    'Sende {itemCount, plural, one {Anlagegut} other {Vermögenswerte}} mit Gebühr',

So i thought this was some sort of plural operator. Could someone confirm this?

Reply:
I've just tested this and the one and other are indeed the definition of 'one asset' and 'multiple assets'. Tested and worked. I've also changes asset and assets to lowercase. Seems more in style with the Dutch language.

Reply:
@comountainclimber can you share your thoughts on this? I have no idea where this is shown or if it is relevant to translate this. Thanks

Reply:
Screenshots of the testing:

<img width="1312" alt="Schermafbeelding 2021-01-15 om 14 29 48" src="https://user-images.githubusercontent.com/6039482/104732887-26793a80-573e-11eb-92e8-3ee4847eefe4.png">
<img width="1312" alt="Schermafbeelding 2021-01-15 om 14 29 39" src="https://user-images.githubusercontent.com/6039482/104732888-2711d100-573e-11eb-9693-cbbcc05ccb2e.png">

Reply:
This looks good to me (is in the send UX @ixje ) Thanks so much for this contribution @Woodehh and for reviewing @ixje !